### DISCUSSION TITLE
Are externalResources supposed to carry through when clicking the button to open CodeSandbox?

### DISCUSSION THREAD

**Discussion by dylanjha** [codesandbox/sandpack#532] [Q&A]
Link:

https://docs.mux.com/guides/video/mux-player#quickstart

This sandpack looks like:

```jsx
import { Sandpack } from '@codesandbox/sandpack-react';

<Sandpack
  options: { externalResources: [`https://unpkg.com/@mux/mux-player`] },
  files={{
    "/index.html": {
      code: `
        <script src="https://unpkg.com/@mux/mux-player"></script>
        <mux-player
          stream-type="on-demand"
          playback-id="EcHgOK9coz5K4rjSwOkoE7Y7O01201YMIC200RI6lNxnhs"
          metadata-video-title="Test video title"
          metadata-viewer-user-id="user-id-007"
        ></mux-player>`.trim(),
      active: true,
    },
    "/src/index.js": {
      code: "",
      hidden: true,
    },
  }}
/>
```

<img width="1016" alt="html-element_2022-07-13_10-18-43" src="https://user-images.githubusercontent.com/764988/178793029-135b7525-b8c3-44d0-bed5-11ec62d8204d.png">

The sandpack itself works great, but then when clicking the icon to open up a new CodeSandbox it seems to be losing the `externalResources`.

**Comment by davekiss** [comment]
Is this possibly an oversight in Sandpack? I'm not seeing the `customResources` returned here:

https://github.com/codesandbox/sandpack/blob/469acb9d610e6d7be06811689c65e46ec791cf11/sandpack-react/src/utils/sandpackUtils.ts#L24-L104

**Comment by danilowoz** [comment]
Unfortunately, our API doesn't support `externalResources`, yet. I'm going to address this internally and make it available on Sandpack as soon as possible.

Thanks for reporting this issue

**Reply by goldenraphti** [reply]
Hi @danilowoz ,
any update on this?
as of end of January 2023 this issue is still present for me

Thanks a lot for all the hard work!
